initfiles/componentfiles/configxml/dali.xsd

```diff
@@ -374,16 +374,16 @@
         </xs:restriction>
       </xs:simpleType>
     </xs:attribute>
-    <xs:attribute name="authMethod" use="optional" default="kerberos">
+    <xs:attribute name="authMethod" use="optional" default="">
```

Comment:
Same as above, the authMethod attribute probably should be removed (please test dali deployment after the removal to make sure nothing breaks...)